clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceServerConfig.java

```diff
@@ -1076,6 +1078,8 @@ public VeniceServerConfig(VeniceProperties serverProperties, Map<String, Map<Str
     this.isParticipantMessageStoreEnabled = serverProperties.getBoolean(PARTICIPANT_MESSAGE_STORE_ENABLED, false);
     idleIngestionTaskCleanupIntervalInSeconds =
         serverProperties.getInt(SERVER_IDLE_INGESTION_TASK_CLEANUP_INTERVAL_IN_SECONDS, -1);
+    useHeartbeatLagForReadyToServeCheckEnabled =
```

Comment:
Let's get a ticket somewhere to deprecate the option to even opt out of this.  Naturally we should keep it configurable until it's deployed and verified to be ok, but I think it'd be a good clean up action to follow.

Reply:
Done